.github/scripts/release.sh

```diff
@@ -9,32 +9,18 @@ if [[ $CURRENT_BRANCH != $PRODUCTION_BRANCH ]]
   exit 1
 fi
 
-RELEASE_OPTION=$1
-case $RELEASE_OPTION in
-  --major)
-    RELEASE_TYPE='major'
-    ;;
-  --minor)
-    RELEASE_TYPE='minor'
-    ;;
-  --patch)
-    RELEASE_TYPE='patch'
-    ;;
-esac
+npx standard-version --dry-run "$@"
 
-# If we've specified a type of release, we need to include the `releaseAs` argument
-SPECIFIED_RELEASE_TYPE_ARG=$([ $RELEASE_TYPE ] && echo "--releaseAs $RELEASE_TYPE");
-CURRENT_PACKAGE_VERSION=$(node -e "process.stdout.write(require('./package.json').version)")
-NEXT_VERSION=$(npx next-standard-version ${SPECIFIED_RELEASE_TYPE_ARG})
-echo "\033[33mBump plugin from ${CURRENT_PACKAGE_VERSION} to ${NEXT_VERSION}?\033[0m\n"
```

Comment:
Since it'll no longer nicely format the new version, it might be nice to add to the documentation that when running release, it'll run a dry run that will output the next version. The output from `--dry-run` doesn't make it _really_ obvious what's going on for those who haven't used Standard Version before.

Reply:
Yeah, I agree it's not ideal. Do you think it would help to reduce the amount of output by skipping some options? I went ahead and pushed a commit for this. If you want to check it out and give me your thoughts. It now looks like this when running a release:
```sh
❯ npm run release

> @sparkbox/sparkle@1.4.4 release
> ./.github/scripts/release.sh

Pending version change:

✔ bumping version in package.json from 1.4.4 to 1.4.5
✔ bumping version in package-lock.json from 1.4.4 to 1.4.5

Nothing commited yet...

Press "y" to proceed with this release or press any other key to abort.
```

Reply:
Yeah - this looks much clearer to me personally.

Reply:
@jordanleven if you are good with this, would you mind approving? Or @zastrow if you have other thoughts, please share. 😄